### PR TITLE
Assets cleanup

### PR DESCRIPTION
**Pull request by lucatume**
This PR removes some assets that are not connected to scripts anymore.

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ class Tribe__Events__Main {
 		 *
 		 * @deprecated 4.8
 		 */
-		const MIN_COMMON_VERSION  = '6.4.1-dev';
+		const MIN_COMMON_VERSION  = '5.2.7-dev';
```
[the remark this answers is not in the document]
Comment (Camwyn) [reply src/Tribe/Main.php:57]:
```suggestion
		const MIN_COMMON_VERSION = '5.2.7-dev';
```

File: .puprc [hunk 2]
```diff
@@ -1,28 +1,19 @@
...
 		"npm ci",
 		"npm run build"
 	],
 	"checks": {
 		"tbd": {
-			"dirs": [
-				"src",
-				"common/src"
-			]
+			"dirs": [ "src" ]
```
Comment (borkweb) [inline .puprc:16]: I'd undo this change, so that when packaging TEC it can control all TBD detection in both itself and common.

File: .puprc [hunk 3]
```diff
@@ -66,17 +43,14 @@
...
 			{
 				"file": "readme.txt",
 				"regex": "(Stable tag: )(.+)"
 			}
 		],
-		"views": [
-			"src/views",
-			"common/src/views"
-		]
+		"views": [ "src/views" ]
```
Comment (borkweb) [inline .puprc:53]: I'd undo this change - the goal is for version checking to be done with templates during packaging, so this will need to remain (though, I haven't ported that `tut` command over yet).

**Review by borkweb — CHANGES_REQUESTED** [review]

File: src/Tribe/Assets.php [hunk 4]
```diff
@@ -40,7 +40,6 @@ public function register() {
 			$plugin,
 			[
 				[ 'jquery-placeholder', 'vendor/jquery-placeholder/jquery.placeholder.js', [ 'jquery' ] ],
-				[ 'tribe-events-php-date-formatter', 'vendor/php-date-formatter/js/php-date-formatter.js', [] ],
```
Comment (Camwyn) [inline src/Tribe/Assets.php:43]: This is still used in ECP
